### DISCUSSION TITLE
Why in the config folder there are two AcConstants file?

### DISCUSSION THREAD

**Discussion by SMorettini** [nasa/fprime#1659] [Q&A] [answered]
I notice that in the `config` folder there is `AcConstants.ini` and `AcConstants.fpp`. They contain exactly the same constant definitions. 

Why do we have both? Should we not have only one?

**Comment by LeStarch** [comment, marked as answer]
Yes.  https://github.com/nasa/fprime/issues/1579

In short, we need to fix that one!
